ecr.tf

```diff
@@ -0,0 +1,51 @@
+# Create secret in AWS SecretsManager to store credentials for upstream repo
+resource "aws_secretsmanager_secret" "ecr_pullthroughcache_dspacecloudreleases" {
+  count                   = var.enable_ecr_pullthrough_rule ? 1 : 0
+  name                    = "ecr-pullthroughcache/dspacecloudreleases"
+  recovery_window_in_days = 7
+}
+
+# Store data inside already created secret
+resource "aws_secretsmanager_secret_version" "ecr_credentials" {
+  count         = var.enable_ecr_pullthrough_rule ? 1 : 0
+  secret_id     = aws_secretsmanager_secret.ecr_pullthroughcache_dspacecloudreleases[0].id
+  secret_string = jsonencode(var.registry_credentials)
```

Comment:
This could be a good pattern for other pull-through cache secrets, but since this is a public registry, I would hard-code dummy values here.

Reply:
They are hard-coded inside variables.tf, as default value (ie empty strings / dummy data)